DataDog/dd-continuous-profiler-example
# Fix and modernize movies-api-java

**What does this PR do?**

This PR includes a number of small changes (see commit-by-commit for details) to bring `movies-api-java` back to life, allowing me to run through https://docs.datadoghq.com/getting_started/profiler/ again.

This included moving to a modern gradle version, moving to a modern JDK image, and a few more tiny details.

**Motivation:**

Fix bitrotten service :)

**Additional Notes:**

N/A

**How to test the change?**

Run through https://docs.datadoghq.com/getting_started/profiler/ and see if you can successfully get to the end.
